.github/workflows/lint-checker.yml

```diff
@@ -64,10 +64,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: setup go
-        uses: actions/setup-go@v1
+      - name: Setup Go Env
+        uses: actions/setup-go@v2
         with:
-          go-version: '1.13'
+          go-version: "1.16.8"
```

Comment:
I think we should use the latest patch version of go 1.16 version, as there are some security fixes after go 1.16.8.

> go1.16.9 (released 2021-10-07) includes a security fix to the linker and misc/wasm directory, as well as bug fixes to the runtime and to the text/template package. See the Go 1.16.9 milestone on our issue tracker for details.

> go1.16.10 (released 2021-11-04) includes security fixes to the archive/zip and debug/macho packages, as well as bug fixes to the compiler, linker, runtime, the misc/wasm directory, and to the net/http package. See the Go 1.16.10 milestone on our issue tracker for details.

> go1.16.11 (released 2021-12-02) includes fixes to the compiler, runtime, and the net/http, net/http/httptest, and time packages. See the Go 1.16.11 milestone on our issue tracker for details.